### DISCUSSION TITLE
Move to balance model?

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#113] [Large CIPs]
I noticed mentions of the move to balance model by @redpartyhat 

> This will be a manageable and temporary increase in storage that will be relieved when CLOUT is moved from UTXOs to a balance model.

https://github.com/bitclout/cips/pull/109

And @diamondhands0  here:

https://github.com/bitclout/cips/pull/109#discussion_r684595111

> But even then it's not a big deal because we'll move to balance model shortly.

And in the code here:

https://github.com/bitclout/core/pull/63/commits/b823978c788d0eb5404febdbdb3837675ca3c43e#diff-7fc2e28969d594902d8507b410e849b6feef5e285e57162f5b81f5c26241fca3R479

```go
type PGBalance struct {
 	tableName struct{} `pg:"pg_balances"`

 	PublicKey    *PublicKey `pg:",pk,type:bytea"`
 	BalanceNanos uint64     `pg:",use_zero"`
 }
```

As this is a protocol change - should this not be a CIP?

Specifically I would be interested to understand:
- will UTXOs be removed completely?
- pros & cons
- impact on scalability
- risks for community developed tools

I can see positives for sure - eg it will be easier accounting, easier to determine total clout in circulation, and reduce size of the chain.

**Comment by kuririn1** [comment]
I would love change to the balance model. UTXOs are hard to read and take a lot of space in the calls (for example mempool call).

**Comment by tijno** [comment]
Looks like this has moved to DIP?

https://github.com/deso-protocol/dips/pull/172
